app/start_handlers.py

```diff
@@ -99,6 +99,7 @@ async def get(self, request):
                     'page_url': View.gen_page_url(request)
                 }
         except KeyError:
+            logger.info('adlocation error')
```

Comment:
Log text should be 'no adlocation present' - the KeyError happens when there is no adlocation provided, which is the normal behaviour